xbmc/music/infoscanner/MusicInfoScanner.cpp

```diff
@@ -919,6 +925,65 @@ int CMusicInfoScanner::RetrieveMusicInfo(const std::string& strDirectory, CFileI
   return numAdded;
 }
 
+void CMusicInfoScanner::CheckBoxSets(VECALBUMS& albums)
+{
+  /* Strategy: Having scanned the tags of the albums to add to the library,
+     check for disc subtitles. If an album only has one subtitled disc, we don't
+     consider it a boxset as its most likely a 'live'version of the album or a 'bonus disc'.
```

Comment:
Sorry for being a pedant
```suggestion
     consider it a boxset as it's most likely a 'live' version of the album or a 'bonus disc'.
```

Reply:
Fixed.